server/controllers/auth.controller.js

```diff
@@ -190,6 +193,38 @@ function register(req, res, next) {
   // also so no-one can have the same email or same username.
   const userQuery = email ? queryIfEmail : queryIfEmailMissing;
 
+  // Sign up user for mailchimp list (if checked)
+  console.log(`newsletter status:${newsletterSignup}`);
+  if (newsletterSignup) {
+    const postData = JSON.stringify({ status: 'subscribed', email_address: email });
+    // Build route because it varies based on API key
+    const hostname = `${config.mailchimp.mailchimpKey.split('-')[1]}.api.mailchimp.com`;
```

Comment:
Wondering if we should add a try catch around this entire if(newsLetterSignup) since if it fails we might miss the registration?